localization-pack-kr/StringTableKrUI.xml

```diff
@@ -728,20 +728,112 @@
             <GameDBLocalizedString>      <LocID>STATISTICS_PATIENTS</LocID>                 <Text>환자 (전체 환자/치료된 환자)</Text>  </GameDBLocalizedString>
 
             <!-- ============= NEW, NEED TRANSLATING ================== -->
-            <GameDBLocalizedString>      <LocID>TOOLTIP_EVENTS_UNLOCK</LocID>               <Text>당신이 모든 부서의 입원실을 열 때 이벤트가 해금될 것입니다.</Text>  </GameDBLocalizedString>
-            <GameDBLocalizedString>      <LocID>TOOLTIP_INCREASE_LOAN</LocID>               <Text>대출금 $20 000.</Text>  </GameDBLocalizedString>
-            <GameDBLocalizedString>      <LocID>TOOLTIP_DECREASE_LOAN</LocID>               <Text>상환금 $20 000.</Text>  </GameDBLocalizedString>
-            <GameDBLocalizedString>      <LocID>HIRING_AGENCY</LocID>                       <Text>에이전시 고용하기</Text>  </GameDBLocalizedString>
-            <GameDBLocalizedString>      <LocID>MANAGEMENT_INSURANCE_HOSPITALIZED</LocID>   <Text>보험 (입원)</Text>  </GameDBLocalizedString>
-            <GameDBLocalizedString>      <LocID>MANAGEMENT_WAGES_HOSPITALIZED</LocID>       <Text>임금 (입원)</Text>  </GameDBLocalizedString>
-            <GameDBLocalizedString>      <LocID>MANAGEMENT_BUDGET_BUILDING</LocID>          <Text>빌딩</Text>  </GameDBLocalizedString>
-            <GameDBLocalizedString>      <LocID>MANAGEMENT_BUDGET_EVENTS</LocID>            <Text>이벤트</Text>  </GameDBLocalizedString>
-            <GameDBLocalizedString>      <LocID>MANAGEMENT_BUDGET_AMBULANCES</LocID>        <Text>구급차</Text>  </GameDBLocalizedString>
-            <GameDBLocalizedString>      <LocID>MANAGEMENT_BUDGET_OTHER</LocID>             <Text>기타</Text>  </GameDBLocalizedString>
-            <GameDBLocalizedString>      <LocID>GET_GREAT_INTERNS</LocID>                   <Text>다음 {1} 인턴이 고용가능할 것 입니다. 훌륭한 지원자가 되겠군요!</Text>  </GameDBLocalizedString>
-            <GameDBLocalizedString>      <LocID>EVENT_BONUS</LocID>                         <Text>이벤트 보너스:</Text>  </GameDBLocalizedString>
-            <GameDBLocalizedString>      <LocID>MONEY_BONUS</LocID>                         <Text>보너스 예산:</Text>  </GameDBLocalizedString>
-
+            <GameDBLocalizedString>      <LocID>TOOLTIP_EVENTS_UNLOCK</LocID>               <Text>Events will unlock when you open hospitalization on all departments.</Text>  </GameDBLocalizedString>
+            <GameDBLocalizedString>      <LocID>TOOLTIP_INCREASE_LOAN</LocID>               <Text>Loan $20 000.</Text>  </GameDBLocalizedString>
```

Comment:
You 'translating' the Korean text back into English here. Is this on purpose?

Reply:
Yes it is on purpose. Because when they update new lines they mixed up some of line number. So I decided to do not update until UI file is finished.  Also I have the original translated script by myself and keep updating. So there is no problem.

Reply:
Alright. Perfect. I'll merge it in, then.